doc/app/Generic_AT_commands.rst

```diff
@@ -101,12 +100,16 @@ The following command example reads the versions:
 
 ::
 
-   AT#XSLMVER
-   #XSLMVER: "2.5.0","2.5.0-lte-5ccd2d4dd54c"
+   AT#XSMVER
+   #XSMVER: "v0.1.0","3.1.99"    // Released build
```

Comment:
Is there some difference in how NCS retrieve the version as it doesn't have `v` prefix?

Reply:
And as our version is directly from Git, it means that `v0.1.0-73-g0c2af8c` points to exactly one specific Git commit so the build is repeatable.

But NCS version `3.1.99` is not repeatable. It does not specify a Git hash.

Reply:
So we'll go ahead with `v`.